common/src/main/java/com/redhat/cloud/notifications/openbridge/BridgeHelper.java

```diff
@@ -73,8 +78,12 @@ public Bridge getBridgeIfNeeded() {
         try {
             bridgeList = apiService.getBridgeByName(ourBridgeName, token);
         } catch (WebApplicationException e) {
+            String path = "GET " + BASE_PATH + "?name";
+            rhoseErrorMetricsRecorder.record(path, e);
             if (e.getResponse().getStatus() == 404) {
                 Log.errorf("Bridge with name %s not found in the OpenBridge instance. Did you create it?", ourBridgeName);
+            } else {
+                throw e;
```

Comment:
This introduce a change of the app behavior. The exception was swallowed before, which I think wasn't a good thing.